src/components/calcite-checkbox/calcite-checkbox.scss

```diff
@@ -1,21 +1,18 @@
 // scale
 :host([scale="s"]) {
-  --calcite-checkbox-size: 12px;
+  --calcite-checkbox-size: theme("spacing.3");
 }
 :host([scale="m"]) {
-  --calcite-checkbox-size: 16px;
+  --calcite-checkbox-size: theme("spacing.4");
 }
 :host([scale="l"]) {
-  --calcite-checkbox-size: 20px;
+  --calcite-checkbox-size: theme("spacing.5");
 }
 ::slotted(input) {
-  display: none;
+  @apply hidden;
 }
 :host {
-  display: inline-flex;
-  cursor: pointer;
-  position: relative;
-  user-select: none;
+  @apply inline-flex cursor-pointer relative select-none;
```

Comment:
These are the ones I have questions about - since they aren't helping enforce consistency to our custom values, and actually add a level of abstraction from the actual css value (ie `select-none` vs: `user-select:  none`), I wonder if we should use `@apply` as alias for single css values? Open to 💬 there.

Reply:
Offline discussion - Yes, this should also work:

```css
.foo {
  @apply
    inline-flex
    cursor-pointer
    relative
    select-none
}
```
This is useful if you're diff viewer does not support multi-level diff viewing.